src/igvfd/types/sample.py

```diff
@@ -74,6 +74,12 @@ class Sample(Item):
         'sample_terms'
     ]
 
+    set_status_up = [
+        'construct_library_sets',
```

Comment:
add documents

Reply:
add cell fate protocol as well

Reply:
Cell fate change protocol is included only in the additional list of In Vitro System (line 494). I didn't want to include it in the parent Sample class since the property is specific to In Vitro System but I can also move it to Sample if that is simpler. What would make more sense?

I will add Documents to the Sample list.